model/validation.go

```diff
@@ -669,3 +661,89 @@ func ValidateDestinationPolicy(msg proto.Message) error {
 
 	return errs
 }
+
+func validateProxyAddress(hostAddr string) error {
+	colon := strings.Index(hostAddr, ":")
+	if colon < 0 {
+		return fmt.Errorf("':' separator not found in %q, host address must be of the form <DNS name>:<port> or <IP>:<port>",
+			hostAddr)
+	}
+	port, err := strconv.Atoi(hostAddr[colon+1:])
+	if err != nil {
+		return err
+	}
+
+	if err = validatePort(port); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateDuration(pd *duration.Duration) error {
+	dur, err := ptypes.Duration(pd)
+	if err != nil {
+		return err
+	}
+	if dur <= 0 {
```

Comment:
upper and lower bounds.
drain/parent shutdown duration: >1s <1hr?
refresh delay: >1s <10m ?
connect timeout: >1ms <30s